packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -92,6 +92,13 @@ of the iOS app that is using this module.''',
         abbr: 'n',
         negatable: false,
         help: 'Validate but do not upload the patch.',
+      )
+      ..addOption(
+        'private-key-path',
+        hide: true,
+        help: '''
+The path for a private key file that will be used to sign the patch artifact.
+''',
```

Comment:
We should (now or at least before this is announced) add more info about what format this file should be in, how to generate it, etc. Could be a link to docs.